### PR TITLE
module 1 task

### PR DESCRIPTION
**Pull request by ghost**
@szikszail I updated little bit branches and reopened MRs)

### CODE REVIEW

File: module-1/classification.js [hunk 1]
```diff
@@ -23,8 +24,22 @@ function grade(score) {
...
+    }
+    if (score <= 59) {
+        gradeOfStudent = 1;
+    } else if (score <= 69) {
+        gradeOfStudent = 2;
+    } else if (score <= 79) {
+        gradeOfStudent = 3;
+    } else if (score <= 89) {
+        gradeOfStudent = 4;
+    } else if (score <= 100) {
```
Comment (szikszail) [inline module-1/classification.js]: I think you don't need this last condition, here score is [90;100]

**Review by szikszail — CHANGES_REQUESTED** [review]
Please check my comments.

**Comment by ghost** [comment]
Changed the last statement to just else.

File: module-1/fibonacci.js [hunk 2]
```diff
@@ -17,7 +17,14 @@ function fibonacci(n) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    let secondVal;
+    if (n < 0) return 0;
+    let arr = [0, 1];
+    for (var i = 1; i <= n; i++) {
```
Comment (szikszail) [inline module-1/fibonacci.js:23]: I think here you should start from 2, since you already have the value for 1

**Review by szikszail — CHANGES_REQUESTED** [review]

**Comment by szikszail** [comment]
Thank you for your solution!

**Pull request closed without merging**